docker/Dockerfile

```diff
@@ -175,6 +175,20 @@ ENV PG_VERSION_MAJOR=${PG_VERSION_MAJOR}
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c", "-e"]
 
+# Install Postgis
+ENV POSTGIS_MAJOR 3
+ENV POSTGIS_VERSION 3.4.2+dfsg-1.pgdg120+1
+
+RUN apt-get update \
```

Comment:
We already install ca-certificates below, otherwise this looks okay to me. Can we consolidate the run commands to reduce the number of layers, perhais with the `ca-certificats` step below?